src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -74,6 +74,11 @@ public class AdminController : Controller
 
         public async Task<IActionResult> Index(TenantIndexOptions options, PagerParameters pagerParameters)
         {
+            if (!await _authorizationService.AuthorizeAsync(User, Permissions.ManageTenants))
+            {
+                return Forbid();
+            }
+
```

Comment:
can you add this check in as well, (we do it on the other methods) so I'm guessing it's relevant
```
            if (!IsDefaultShell())
            {
                return Forbid();
            }
```